vimeo-networking/src/main/java/com/vimeo/networking/VimeoClient.java

```diff
@@ -422,6 +419,36 @@ public VimeoAccount authorizeWithClientCredentialsGrantSync() {
         return call;
     }
 
+    /**
+     * Synchronous version of {@link #singleSignOnTokenExchange(String, AuthCallback)}. This is useful
+     * when already running in a background thread, such as when using the Android AccountManager.
+     *
+     * @param token           the authenticated user access token from application A. This <b>cannot</b> be a client
```

Comment:
It just means that it must be an authenticated user - user is the key word here. Adding more specificity here 99d20cf